crates/store/re_video/src/decode/mod.rs

```diff
@@ -88,6 +88,8 @@ mod webcodecs;
 
 use crate::Time;
 
+pub use ffmpeg::Error as FfmpegError;
```

Comment:
```suggestion
#[cfg(with_ffmpeg)]
pub use ffmpeg::Error as FfmpegError;
```

…or revert the changes to this file